Formula/babl.rb

```diff
@@ -13,16 +13,24 @@ class Babl < Formula
     sha256 "3cd4d6d3cc86dc1bc2b8411bf4ffb911df58458d54fc938730fdede11587c624" => :sierra
   end
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
   depends_on "libtool" => :build
+  depends_on "little-cms2" => :build
```

Comment:
```suggestion
  depends_on "little-cms2"
```

`little-cms2` is a runtime dependency. Place it after `gobject-introspection`